idl-compiler.py

```diff
@@ -543,15 +543,9 @@ def send_function_signature_params_list(self, include_placeholder_names):
         return res
 
     def send_message_argument_list(self):
-        res = f'ms, '
-        if self.with_timeout and self.one_way:
-            # For some reason the timeout argument position in
-            # `send_message_oneway_timeout` is different from `send_message_timeout`.
-            res += f'timeout, {self.messaging_verb_enum_case()}, id'
-        else:
-            res += f'{self.messaging_verb_enum_case()}, id'
-            if self.with_timeout:
-                res += ', timeout'
+        res = f'ms, {self.messaging_verb_enum_case()}, id'
+        if self.with_timeout:
+            res += ', timeout'
```

Comment:
Please explain in the commit message how this builds up to what you're trying to accomplish.

Reply:
It's called "boy scout refactoring". Simplifying this logic makes writing later code easier. I didn't need to do that, I could follow the existing "design" and when adding new stuff write a 4-level deep tree of ifs. Should I explain that in the commit message?